securedrop/sass/_source_index.sass

```diff
@@ -31,22 +36,60 @@
 
     .index-column
       padding: 1em
-      display: block
+      display: flex
+      flex-direction: column
+      justify-content: space-between
       width: 48%
-      border: 1px solid $color_purple_medium
       border-top-width: 0
       border-bottom-width: 0
 
     .index-left
-      background: $color_purple_medium
+      background-image: linear-gradient(140deg, #0096dc, #005db7)
```

Comment:
would using variables for theses colors make sense here, since they are reused later?